src/framework/audio/audiotypes.h

```diff
@@ -45,6 +45,7 @@ using volume_db_t = float;
 using volume_dbfs_t = float;
 using gain_t = float;
 using balance_t = float;
+using audio_signal_amount_t = float;
```

Comment:
use gain_t instead, technically it's really what we're using there

Reply:
I'll fix this in the next PR